### PR TITLE
human play mode

### PR DESCRIPTION
**Pull request by yotambarzilay**

### CODE REVIEW

File: src/App.svelte [hunk 1]
```diff
@@ -15,25 +15,114 @@
...
-    let state = createInitialState()
+    let mode = 'human'
+
+    $: {
+        if (mode) { // reset when mode changes
+            state = createInitialState()
+        }
+    }
 
+    let pendingMoves = null
```
Comment (kfiroo) [inline src/App.svelte]: not used

File: src/App.svelte [hunk 2]
```diff
@@ -15,25 +15,114 @@
         playGame
 	} from './game'
 
-    let state = createInitialState()
+    let mode = 'human'
+
+    $: {
```
Comment (kfiroo) [inline src/App.svelte]: what is this magic?

File: src/App.svelte [hunk 3]
```diff
@@ -15,25 +15,114 @@
...
+    const isValid = position => !!_.find(moves, ({index, index1, index2}) =>
+        index === position || index1 === position || index2 === position
+    )
+
+    const onBoardClick = ({currentTarget}) => {
+        if (state.phase === GAME_OVER || state.phase === WINNER) {
+	        return
+        }
+        
+        const { target } = event
```
Comment (kfiroo) [inline src/App.svelte]: what is?

File: src/App.svelte [hunk 4]
```diff
@@ -15,25 +15,114 @@
...
+        const position = parseInt(currentTarget.dataset.position, 10)
+        const card = state.board[position]
+        if (state.phase === REMOVE_CARDS) {
+            if (!card) {
+                return
+            }
+
+            if (parseInt(card.value, 10) === 10) {
```
Comment (kfiroo) [inline src/App.svelte]: have getCardValue

File: src/App.svelte [hunk 5]
```diff
@@ -15,25 +15,114 @@
...
+            })
+
+            if (action) {
+                state = playTurn(state, action)
+            }
+        }
+    }
+
+    const img = new Image()
+    img.src = 'https://dok7xy59qfw9h.cloudfront.net/587/070/202/-239996995-1t62joi-8mq5akftktd5se5/original/file.jpg'
```
Comment (kfiroo) [inline src/App.svelte]: betah preload

**Comment by kfiroo** [comment]
You are a genius monkey

File: src/App.svelte (hunk 2 above)
Comment (yotambarzilay) [reply src/App.svelte]: reactive side-effect. it will be called whenever mode is changed

File: src/App.svelte (hunk 3 above)
Comment (yotambarzilay) [reply src/App.svelte]: is because no prettier

**Pull request merged**